### ISSUE TITLE
I can not access pages with browser once running with network commands

### ISSUE THREAD

**Issue by k4route** [k4route/learn_docker#1]
When spinning up containers with custom network browser says not found to index.php. Something wrong with interconnection between nginx and php. Need to troubleshoot.

Should be seen:

Hostname 'redis' can be found at: 172.17.0.2
Connected to the database at hostname 'mysql': 172.17.0.3

**Comment by k4route** [comment]
Please create branch Issue_1 and investigate. Once fixed perform pull request.

**Comment by k4route** [comment]
On the moment of second check everything went well actually and I was able to get to image php:

Hostname 'redis' can be found at: 10.44.0.2
Connected to the database at hostname 'mysql': 10.44.0.3

I have found out mysql creates unique image every time it runs. So now have to clean up and define named image for every run.

local     02d65da88cebbb1d0ead1fd3bca3e76d660ffe366ce81ca9f9dcb685fe109c5c
local     8f0e6d52bbfd9181e1aaebd650ce36027b7b92df9a1769f147c8ddf3ba2c6a8b
local     3735f9b04983b78e175929a666bae182b5676369ae027fd82d4d6233b507b801
local     9524290d036edc52fe14a979b4b5359f1310790fc1eab038b2c45b103096329d
local     b085b50bf1da58cdb435ce374b8ca3ed996ab5c0eb861ec9bda7a3416dfcb5cd
local     bbf6cacd2e2c2be6e1eca5f0f4383502c876eb7161485253571174a4f419951c
local     c9e35acaf7faca098828ab6aeb491dc0710aff6637264dd95c3eaaf02a7517f5
local     d04e26e9a9fb6a9b5325a113bab02a2c1d672310b995b9a08119fa3a6c45ddec
local     dac28ae888d0cd63750ea9aac4f45868f7a1e384003677dce90996a2bc0c524d
local     dce690a97a4dd84714798906712bb3864c315c22f363c249aeac7c06c925657e

redis also should create an image every time.
